mempool/src/error.rs

```diff
@@ -81,6 +81,8 @@ pub enum TxValidationError {
     RollingFeeThresholdNotMet { minimum_fee: Amount, tx_fee: Amount },
     #[error("Overflow encountered while updating ancestor fee.")]
     AncestorFeeUpdateOverflow,
+    #[error("Fee overflow")]
+    FeeOverflow,
```

Comment:
I'm just curious if we can have a more specific error (such as `AncestorFeeUpdateOverflow` above) or there are several completely different places where the new error can occur?..

Reply:
The new error occurs in just one place. But calling it `FeePrecisionMultiplicationOverflow` or something of the sort seemed over the top....